src/Fixer/PhpUnit/PhpUnitNoExpectationAnnotationFixer.php

```diff
@@ -275,10 +276,10 @@ private function extractContentFromAnnotation(Annotation $annotation)
     private function annotationsToParamList(array $annotations)
     {
         $params = [];
-        $exceptionClass = '\\'.ltrim($annotations['expectedException'], '\\');
+        $exceptionClass = ltrim($annotations['expectedException'], '\\');
 
         if ($this->configuration['use_class_const']) {
-            $params[] = $exceptionClass.'::class';
+            $params[] = '\\'.$exceptionClass.'::class';
```

Comment:
Can we have `sprintf` here?

Reply:
I would even write `"\\{$exceptionClass}::class"`.

Reply:
If you can find a resolution would be great. I can't write both in there at the same time.

IMHO neither of the three is of any difference. The projects PHP-CS-Fixer configuration suggests `"\\${exceptionClass}::class"`for string interpolation, but only unless the variant @julienfalque suggested is used.

So if you come to a conclusion, I'm happy to file it:

* A; `'a'.$b.'c'` style (as originally filed) 
* B: sprintf() call
* C: `"{$a}efgh"` style
* D: `"${a}efgh"` style

From the existing style in the file I prefer C, but please let me know which is the general preference in the project as I think this is worth to have at hand directly.

Reply:
Myself concludes as even after a day, no result of the poll was given by others, that the resolution C is acceptable for this PR.

Thanks a lot for putting your efforts into the review, I really like to have multiple reviewers on a pull request and I know that this is nothing to expect.